lnwallet/fee_estimator.go

```diff
@@ -297,15 +301,30 @@ func (b *BitcoindFeeEstimator) EstimateFeePerWeight(numBlocks uint32) (btcutil.A
 // fetchEstimate returns a fee estimate for a transaction be be confirmed in
 // confTarget blocks. The estimate is returned in sat/byte.
 func (b *BitcoindFeeEstimator) fetchEstimatePerByte(confTarget uint32) (btcutil.Amount, error) {
-	// First, we'll fetch the estimate for our confirmation target.
-	btcPerKB, err := b.bitcoindConn.EstimateFee(int64(confTarget))
+	// First, we'll send an "estimatesmartfee" command as a raw request,
+	// since it isn't supported by btcd but is available in bitcoind.
+	target, err := json.Marshal(uint64(confTarget))
+	if err != nil {
+		return 0, err
+	}
+	resp, err := b.bitcoindConn.RawRequest("estimatesmartfee",
```

Comment:
Can you add a TODO here to later introduce usage of the economical/conservative modifiers of the PRC call? Thanks!